docs/coding-guidelines/adding-api-guidelines.md

```diff
@@ -4,15 +4,12 @@ Recommended reading to better understand this document:
 | [Package-Projects](package-projects.md)
 
 # Add APIs
-- [Determining versions and targets](#determining-versions-and-targets)
-- [Making the changes in repo](#making-the-changes-in-repo)
-- [FAQ](#faq)
-
-## Determining versions and targets
-
-1. [Determine what library](#determine-what-library) the API goes into.
-2. [Determine the target framework](#determine-target-framework) for the library that will contain the API.
-3. [Determine the version](#determine-library-version) for the library that will contain the API.
+- [Add APIs](#add-apis)
+    - [Determine what library](#determine-what-library)
+    - [Determine target framework](#determine-target-framework)
+    - [Determine library version](#determine-library-version)
+  - [Making the changes in repo](#making-the-changes-in-repo)
+  - [FAQ](#faq)
 
 ### Determine what library
```

Comment:
Fix [MD022](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines).

```suggestion
### Determine what library

```